api/RTCStatsReport.json

```
@@ -2606,7 +2606,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤73"
+                "version_added": "≤60"
```

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1374465

```suggestion
                "version_added": "56"
```